crates/pop-parachains/src/call/mod.rs

```diff
@@ -166,4 +174,21 @@ mod tests {
 		assert_eq!(encode_call_data(&api, &extrinsic)?, "0x00000411");
 		Ok(())
 	}
+
+	#[tokio::test]
+	async fn construct_sudo_extrinsic_works() -> Result<()> {
+		let extrinsic = construct_extrinsic(
+			"Balances",
+			"transfer_allow_death",
```

Comment:
Whilst it probably doesnt matter, especially in terms of the test, it would be ideal to use one of the force_ dispatchables which actually requires root origin.

Reply:
In this specific test it doesn't really matter because is only constructing the extrinsic,  but changed.